### ISSUE TITLE
Remove the aria-expanded attribute from RichText

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#12523]
Splitting this out from https://github.com/WordPress/gutenberg/issues/2375#issuecomment-418061593

Currently, Gutenberg uses an `aria-expanded` attribute in the `RichText` component. This was originally implemented for the auto-completers suggestions: when a suggestions list opens, the `aria-expanded` changes to `true` so screen readers announce `expanded` to communicate something appeared in the UI.

This implements the [ARIA combobox pattern](https://www.w3.org/TR/wai-aria-1.1/#combobox), which is typically used for input fields with autocomplete suggestions (for example in Gutenberg: the Add New Tag field).

However, the `RichText` component is not used as an ARIA combobox. Currently, it's always an ARIA `textbox` with an `aria-multiline=true` attribute (only exception: the Table block), which is the equivalent of a `<textarea>` element. 

As mentioned in https://github.com/WordPress/gutenberg/issues/2375#issuecomment-418061593 and https://github.com/WordPress/gutenberg/issues/2375#issuecomment-418071516 the `aria-expanded` attribute is not allowed for the role `textbox` and should be removed.

After all, when a suggestions list appears, the results are announced with an audible message `nn results found, use up and down arrows to navigate` so there's already some feedback.
